phpBB/phpbb/files/filespec.php

```diff
@@ -0,0 +1,586 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\files;
+
+use \phpbb\language\language;
```

Comment:
The leading `\` is not necessary, and most of the core files doesn't use it (you do in all `use` statements). Not sure if there is anything in the coding guidelines about this, so feel free to ignore this.